include/multi_blas_helper.cuh

```diff
@@ -206,21 +211,25 @@ namespace quda
       using SpinorW = SpinorX;
 
       constexpr auto arg_known_size = (sizeof(kernel_param<>)                                        // kernel_param parent
-                                       + sizeof(int)                                                 // NYW parameter
-                                       + sizeof(SpinorX[NXZ])                                        // SpinorX array
-                                       + (Functor::use_z ? sizeof(SpinorZ[NXZ]) : sizeof(SpinorZ *)) // SpinorZ array
-                                       + sizeof(Functor)                                             // functor
-                                       + sizeof(dim3)                                                // threads parameter
-                                       + (!Functor::use_w ? sizeof(SpinorW *) : 0)                   // subtract pointer if not using W
-                                       + (Functor::reducer ? sizeof(ReduceArg<device_reduce_t>) : 0) // reduction buffers
-                                       );
+				       + sizeof(int)                                                 // NYW parameter
```

Comment:
clang-format will catch this, but noting that this diff here mostly arises due to using tab indents